simapp/sim_test.go

```diff
@@ -95,30 +103,137 @@ func appStateFromGenesisFileFn(r *rand.Rand, accs []simulation.Account, genesisT
 		// and these keys are never actually used to sign by mock Tendermint.
 		privkeySeed := make([]byte, 15)
 		r.Read(privkeySeed)
+
 		privKey := secp256k1.GenPrivKeySecp256k1(privkeySeed)
 		newAccs = append(newAccs, simulation.Account{privKey, privKey.PubKey(), acc.Address})
 	}
+
 	return genesis.AppState, newAccs, genesis.ChainID
 }
 
 // TODO refactor out random initialization code to the modules
```

Comment:
is there an issue for this ? edit: Did a quick search and I think there isn't

Reply:
No, I removed it. I don't think we need it.